src/main/java/org/fogbowcloud/manager/core/models/linkedlist/Node.java

```diff
@@ -0,0 +1,42 @@
+package org.fogbowcloud.manager.core.models.linkedlist;
+
+import org.fogbowcloud.manager.core.models.orders.Order;
+
+public class Node {
+
+	private Node next;
+	private Node previous;
+	private Order order;
+
+	public Node(){ }
```

Comment:
Do we need this constructor? This will create a Node with an Order null. We assume no element in the list has an Order that is null.